.travis.yml

```diff
@@ -13,8 +13,25 @@ addons:
       - qtdeclarative5-dev
       - libqt5charts5-dev
 
+# https://docs.platformio.org/en/latest/integration/ci/travis.html#travis-ci
+python:
+    - "2.7"
```

Comment:
For real py3 doesn't work?

Reply:
I was just copying the snippet from the link. I'll try to remove this before merging.

Reply:
Yeah it works without this line, removed. Merging. Thanks!